src/careamics/transforms/n2v_manipulate_torch.py

```diff
@@ -122,6 +118,12 @@ def __call__(
         masked = torch.zeros_like(batch)
         mask = torch.zeros_like(batch, dtype=torch.uint8)
 
+        self.rng = (
```

Comment:
Why has this changed? It implies to me that every N2V Manipulate call will be the same, rather than just following the same steps each time (instantiating a new generator with the same seed each time, vs having a seeded generator create once at the beginning).